src/pages/Beta/Stake/DetailModal/EarnedWidget/styled.tsx

```diff
@@ -9,7 +9,7 @@ export const Root = styled(Box)`
   width: 100%;
   position: relative;
   overflow: hidden;
-  height: 280px;
+  height: auto;
```

Comment:
what is the reason to make height auto? Here we need fixed height because of the drawer we are using. all drawer is of fixed height.

Reply:
Having a fixed height, makes the claim button overflow in earned widget.

Reply:
but without fixed height other drawer ui will start breaking. so we need to think of a fix in that way so that we can keep fix height and also fix the issue. may be we can make font size or inner margin little less to accommodate spacing.

Reply:
Okay, got it. Will look into it.

Reply:
Have adjusted it with padding on mobile now.